inc/class-wpseo-meta.php

```diff
@@ -393,7 +418,7 @@ public static function get_meta_field_defs( $tab, $post_type = 'post' ) {
 					return [];
 				}
 
-				$field_defs['schema_page_type']['default'] = WPSEO_Options::get( 'schema-page-type-' . $post_type );
+				$field_defs['schema_page_type']['default_value'] = WPSEO_Options::get( 'schema-page-type-' . $post_type );
```

Comment:
So this change and the one below in line 433 (`$field_defs['schema_article_type']['default_value'] = $default_schema_article_type;`) breaks the `Meta_Fields_Presenter::presenter()` part: 
```
			if ( isset( $meta_field['default'] ) ) {
				$default = \sprintf( ' data-default="%s"', \esc_attr( $meta_field['default'] ) );
			}
```
which means that if I'm reading the code right, it will stop outputting that data-default attribute. 

I'm not sure of the side-effects for that, but we should follow the path of the code to make sure we can do this. 

If we indeed can live without that data-default attribute, lets mention it into the technical choices

Reply:
This part is now removed and default value will be taken from the `default_value` property for all fields.
The reason I changed it here is because all other fields have the default value set in for `default_value` while the schema default value is set with `default`.